README.md

```diff
@@ -25,24 +25,41 @@ via GitHub pull requests or issues.
 Install the XML extension by RedHat. The exention ID is
 [`redhat.vscode-xml`](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-xml).
 
-### Support for GeekoDoc's RELAX NG schema
-
-To let VSCode be aware of GeekoDoc, you need to associate XML files with the
-GeekoDoc schema. Add the following to `settings.json` and save the file in your
-user directory or the project's workspace:
-
-```json
-"xml.fileAssociations": [
-    {
-        "pattern": "**/*.asm.xml",
-        "systemId": "file:///usr/share/xml/docbook/schema/rng/5.2/assemblyxi.rng"
-    },
-    {
-        "pattern": "**/*.xml",
-        "systemId": "file:///usr/share/xml/geekodoc/rng/geekodoc-latest-flat.rng"
-    }
-],
-```
+### Support for RELAX NG schema
+
+To let VSCode be aware of GeekoDoc and DocBook assembly schema, you have two options:
+
+- Insert an XML model definition at the top of the XML file *before* the root
+  element (for example, after `<?xml version="1.0"/>`).
+    - for GeekoDoc files:
+        ```xml
+        <?xml-model href="urn:x-suse:rng:v2:geekodoc-flat"
+            type="application/xml"
+            schematypens="http://relaxng.org/ns/structure/1.0"?>
+        ```
+    - for DocBook assmbly files:
+        ```xml
+        <?xml-model href="https://cdn.docbook.org/schema/5.2/rng/assemblyxi.rng"
+            type="application/xml"
+            schematypens="http://relaxng.org/ns/structure/1.0"?>
```

Comment:
There is a typo in "assmbly". :wink:

I would amend it with an additional entry for DocBook files. That gives us the three important schemas we need (GeekoDoc, DocBook, and DocBook Assemblies):

```suggestion
    - for DocBook assembly files:
        ```xml
        <?xml-model href="https://cdn.docbook.org/schema/5.2/rng/assemblyxi.rng"
            type="application/xml"
            schematypens="http://relaxng.org/ns/structure/1.0"?>

    - for DocBook files:
        ```xml
        <?xml-model href="https://cdn.docbook.org/schema/5.2/rng/docbookxi.rng"
            type="application/xml"
            schematypens="http://relaxng.org/ns/structure/1.0"?>

```